map/gps_track.hpp

```diff
@@ -101,4 +103,7 @@ class GpsTrack final
   bool m_threadExit; // need exit thread
   bool m_threadWakeup; // need wakeup thread
   std::condition_variable m_cv;
+public:
```

Comment:
Public methods are usually above private methods for the class' interface readability.